lottie-ios/Classes/AnimatableLayers/LOTLayerContainer.m

```diff
@@ -162,9 +162,17 @@ - (void)_setImageForAsset:(LOTAsset *)asset {
     } else {
         NSString *imagePath = [asset.assetBundle pathForResource:asset.imageName ofType:nil];
         image = [UIImage imageWithContentsOfFile:imagePath];
-        if(!image) {
+        if (!image) {
             image = [UIImage imageNamed:asset.imageName inBundle: asset.assetBundle compatibleWithTraitCollection:nil];
         }
+
+        if (!image) {
+            // Try to get the image from an xcasset catalogue
+            NSData *data = [[NSDataAsset alloc] initWithName: asset.imageName].data;
```

Comment:
NSDataAsset is available only for iOS 9.0+, but the deployment target is set to iOS 8.0.